wagtail/admin/panels.py

```diff
@@ -317,6 +330,26 @@ def __init__(self, panel, instance, request, form, prefix):
             self.heading = self.panel.heading
             self.help_text = self.panel.help_text
 
+        @property
+        def is_updating(self) -> bool:
+            """
+            The opposite of ``BoundPanel.is_creating``. This property
+            returns ``True`` if the model instance currently being edited has
+            previously been saved to the database.
+            """
+            return (
+                isinstance(self.instance, Page) and self.instance.id
```

Comment:
Previously, I have found that the `instance._state.adding` check doesn't work for pages. I think because the instance are generally assembled in-memory from revision content. That may have been fixed since, though.

Reply:
Since we are moving away from 'detection' in this way, I am marking this as resolved.